### PR TITLE
Fix colony border rendering

### PR DESCRIPTION
**Pull request by Nightenom**
Progress towards #6657

# Changes proposed in this pull request:
- fixes y shifting and color bugs
image

Review please

### CODE REVIEW

**Comment by someaddons** [comment]
just gave it a try, it doesnt seem to render anything for me for some reason: 
image

File: src/main/resources/META-INF/accesstransformer.cfg [hunk 1]
```diff
@@ -52,4 +52,7 @@ public net.minecraft.client.multiplayer.ClientAdvancementManager field_192803_d
 
 public net.minecraft.block.DoorBlock func_196426_b(Lnet/minecraft/world/World;Lnet/minecraft/util/math/BlockPos;Z)V # DoorBlock playsound
 
-public net.minecraft.entity.Entity field_213326_aJ # eyeHeight
\ No newline at end of file
+public net.minecraft.entity.Entity field_213326_aJ # eyeHeight
+
+# vbo draw
```
Comment (marchermans) [inline src/main/resources/META-INF/accesstransformer.cfg:57]: Why would you need this at here?

**Review by marchermans — COMMENTED** [review]
Generally oke. The rendering seems fishy to me with the at?

File: src/main/resources/META-INF/accesstransformer.cfg (hunk 1 above)
Comment (Nightenom) [reply src/main/resources/META-INF/accesstransformer.cfg:57]: I could probably just copy that function but I think using the vanilla is safer cuz of vanilla change
usage: uploading vbo repeatedly without using bufferbuilder every frame

**Review by someaddons — APPROVED** [review]
works now 👍

**Pull request merged**